CODEOWNERS

```diff
@@ -56,6 +56,7 @@
 /bundles/org.openhab.binding.hdanywhere/ @kgoderis
 /bundles/org.openhab.binding.hdpowerview/ @beowulfe
 /bundles/org.openhab.binding.helios/ @kgoderis
+/bundles/org.openhab.binding.heliosventiremote/ @ramack
```

Comment:
no, I have added Venti to differentiate the Helios Ventilation System from the helios binding one line above and the remote to make it clear that it is intended to be connected to the remote control bus. - As there is also a modbus interface translating between a documented modbus and the internal bus. (But I don't wanna by that one :-)

If you have better proposals let me know.

Reply:
Maybe 'heliosventilation'  would be a good choice.

Reply:
For the mentioned modbus interface there is even an OH1.8  binding: https://github.com/bern77/org.openhab.binding.helios/ and I tried to differentiate my one accessing the remote control bus from it. But if you don't have a concern it's fine for me. I also prefer heliosventilation over the somehow artificial current proposal.
Can you recommend an approach to do a rename not to forget something? Or is there just the "rename directory and fix the issues afterwards" and browse over the files to hopefully find it if there is something missing?

Reply:
I would recommend using 'Refactor/Rename' in Eclipse, otherwise you need to rename all packages by hand. Some changes (bom, feature, binding.xml and XXXBindingConstants) need to be done manually.

Reply:
It took many hours - with this and all the struggles I had before, probably more than the time I invested in real development, but its done.

Reply:
Sorry, I thought it was max. an hour or so. Thanks a lot.

Reply:
no problem. Technically it was just a few minutes. Only the struggle with the compile afterwards took the big efforts.